src/vivarium/framework/time.py

```diff
@@ -69,6 +71,12 @@ def __init__(self):
         self.population_view = None
 
     def setup(self, builder: "Builder"):
+        self.step_size_pipeline = builder.value.register_value_producer(
+            "simulant_step_size",
+            source=lambda idx: [pd.Series(self.step_size, index=idx)],
+            preferred_combiner=list_combiner,
```

Comment:
I forget - what does this list combiner do?

Reply:
It returns a list of Series: first the source, and then any mutated versions of the pipeline (which doesn't necessarily have to be the same size as the source)